packages/storybook/stories/va-search.stories.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { VaSearch } from '@department-of-veterans-affairs/web-components/react-bindings';
+import { getWebComponentDocs, propStructure } from './wc-helpers';
+import { generateEventsDescription } from './events';
+
+const searchDocs = getWebComponentDocs('va-search');
+
+export default {
+  title: 'Components/va-search',
+  parameters: {
+    docs: {
+      description: {
+        component: generateEventsDescription(searchDocs),
+      },
+    },
+  },
+};
+
+const Template = ({ label }) => (
+  <VaSearch
+    label={label}
+    onInputBlurEvent={e => console.log(e, 'INPUT BLUR FIRED')}
+    onInputChangeEvent={e => console.log(e, 'INPUT CHANGE FIRED')}
+    onInputClickEvent={e => console.log(e, 'INPUT CLICK FIRED')}
+    onInputKeyDownEvent={e => console.log(e, 'INPUT KEYDOWN FIRED')}
+    onButtonClickEvent={e => console.log(e, 'BUTTON CLICK FIRED')}
+    onButtonFocusEvent={e => console.log(e, 'BUTTON FOCUS FIRED')}
+    onButtonKeyDownEvent={e => console.log(e, 'BUTTON KEYDOWN FIRED')}
+  />
+);
+
+const defaultArgs = {
```

Comment:
@andresriveratoro do we want to tie in: button-text, hide-button, and input-value props to this story

Reply:
Thanks, I forgot to add these in.